contracts/VaultV1.sol

```diff
@@ -12,51 +12,41 @@ contract VaultV1 is ERC4626, Ownable {
     using SafeTransferLib for ERC20;
 
     //the underlying token the vault accepts
-    ERC20 public immutable UNDERLYING;
+    ERC20 public immutable underlying;
     //the strategy address
-    IStrategy public STRATEGY;
-    //the buffer amount (indicates to bot the minimun amount of tokens that can be stored in the vault)
-    uint256 public BUFFER;
+    IStrategy public strategy;
+    //does a strategy exist
+    bool public strategyExists;
 
     // Withdraw locking params
     mapping(address => uint256) public requestedWithdraws;
     mapping(address => uint256) public unlockTime;
-    uint256 public totalRequestedWithdraws;
     uint256 public withdrawWindow = 24 hours;
```

Comment:
This could be made less than 32 bytes and packed with `strategyExists`
It's also never modified so should be `constant`.